src/sdk/auth.js

```diff
@@ -137,6 +136,20 @@ export class OCAuthCore
     {
         return this.tokenManager.getAccessToken();
     }
+
+    getParsedIdToken ()
+    {
+        // return all info in id token
+        const idToken = this.tokenManager.getIdToken();
```

Comment:
should check for undefined here. if idToken doesn't exist yet, it should return undefined. (test it to verify, but I think that's what storage manager would do). we shouldn't pass undefined to parseJwt coz it would throw random string parsing exception. Instead if there isn't a token we can return `{ }` gracefully.